athena-federation-integ-test/src/main/java/com/amazonaws/athena/connector/integ/CloudFormationClient.java

```diff
@@ -77,8 +83,7 @@ private void processCreateStackRequest(CreateStackRequest createStackRequest)
             throws RuntimeException
     {
         // Create CloudFormation stack.
-        AmazonCloudFormation cloudFormationClient = AmazonCloudFormationClientBuilder.defaultClient();
-        CreateStackResult result = cloudFormationClient.createStack(createStackRequest);
+        CreateStackResult result = client.createStack(createStackRequest);
```

Comment:
why did we go from a descriptive name "cloudFormationClient' to a generic name 'client'?

Reply:
I'll change the name.